packages/contracts-bedrock/scripts/go-ffi/differential-testing.go

```diff
@@ -74,8 +74,8 @@ var (
 	}
 )
 
-func main() {
-	args := os.Args[1:]
+func DiffTestUtils() {
+	args := os.Args[2:]
```

Comment:
nit, it would be nice to specifically name `args[0]` to `variant` to be consistent with trie.go